.github/workflows/publish.yml

```diff
@@ -24,6 +24,23 @@ jobs:
         env:
           REF: ${{ github.ref }}
         run: |
+          # we are using ubuntu-latest as our riscv64 cross build machine
+          if [ "${{ matrix.os }}" = ubuntu-latest ]; then
+             APT_INSTALL="sudo apt install -y binfmt-support qemu-user-static"
+             # the following weird statement is here to speed up the happy path
+             # if the default server is responding -- we can skip apt update
+             $APT_INSTALL || { sudo apt update && $APT_INSTALL ; }
+             # we need to build our own version of cross-enabled linuxkit, while upstream is sorting things out
```

Comment:
Do we care about these 80 character line length checks?
FWIW I know that for golang code the limit is significantly larger than 80 characters.

Reply:
Honestly, I'm trying to be a good citizen -- but in a few [rare] cases where it hurts readability (because nesting) I prefer to go against Yetus' wishes. I don't think it hurt us ever @nordmark -- and in general I agree with the advice -- but I'd rather bump it to golang width.